CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Changelog
 
+### 0.1.0
+
+Features
```

Comment:
note that links are not generated by Markdown -- https://github.com/nginxinc/nginx-ingress-operator/blob/dd6d115761bfd72134bd7ea03e52617dea23f5e3/CHANGELOG.md  - both for PR and users

I wonder if we can stick to the old format? with its section headers and entry format plus only mentioning external contributors.  What didn't work?